svgdigitizer/electrochemistry/cv.py

```diff
@@ -283,42 +372,63 @@ def x_label(self):
             ... <svg>
             ...   <g>
             ...     <path d="M 0 200 L 0 100" />
-            ...     <text x="0" y="200">x1: 0 V vs. RHE</text>
+            ...     <text x="0" y="200">E1: 0 V vs. RHE</text>
             ...   </g>
             ...   <g>
             ...     <path d="M 100 200 L 100 100" />
-            ...     <text x="100" y="200">x2: 1 V vs. RHE</text>
+            ...     <text x="100" y="200">E2: 1 V vs. RHE</text>
             ...   </g>
             ...   <g>
             ...     <path d="M -100 100 L 0 100" />
-            ...     <text x="-100" y="100">y1: 0 uA / cm2</text>
+            ...     <text x="-100" y="100">j1: 0 uA / cm2</text>
             ...   </g>
             ...   <g>
             ...     <path d="M -100 0 L 0 0" />
-            ...     <text x="-100" y="0">y2: 1  uA / cm2</text>
+            ...     <text x="-100" y="0">j2: 1 uA / cm2</text>
             ...   </g>
             ...   <text x="-200" y="330">scan rate: 50 V/s</text>
             ... </svg>'''))
             >>> cv = CV(SVGPlot(svg))
-            >>> cv.x_label
-            Label(label='V vs. RHE', unit='V', reference='RHE')
+            >>> cv.figure_schema  # doctest: +NORMALIZE_WHITESPACE
+            {'fields': [{'name': 'E', 'unit': 'V', 'orientation': 'x', 'reference': 'RHE'},
+                        {'name': 'j', 'unit': 'uA / cm2', 'orientation': 'y'}]}
 
-        Label and unit can be obtained by::
+        """
+        import re
 
-            >>> cv.x_label.unit
-            'V'
-            >>> cv.x_label.reference
-            'RHE'
+        schema = self.svgplot.schema
 
-        """
         pattern = r"^(?P<unit>.+?)? *(?:(?:@|vs\.?) *(?P<reference>.+))?$"
         match = re.match(
-            pattern, self.svgplot.axis_labels[self.svgplot.xlabel], re.IGNORECASE
+            pattern, schema.get_field(self.voltage_dimension)["unit"], re.IGNORECASE
         )
 
-        return namedtuple("Label", ["label", "unit", "reference"])(
-            match[0], match[1], match[2] or "unknown"
-        )
+        schema.get_field(self.voltage_dimension)["unit"] = match[1]
+        schema.get_field(self.voltage_dimension)["reference"] = match[2] or "unknown"
+
+        return schema
+
+    @classmethod
+    def get_axis_unit(cls, unit):
+        r"""
+        Return `unit` as an `astropy <https://docs.astropy.org/en/stable/units/>`_ unit.
```

Comment:
If we end up using intersphinx, we could also link to astropy in a stable way here.